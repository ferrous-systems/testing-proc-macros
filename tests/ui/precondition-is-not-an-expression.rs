use contracts::contracts;

#[contracts]
#[precondition(struct)]
fn f() {}

fn main() {}
