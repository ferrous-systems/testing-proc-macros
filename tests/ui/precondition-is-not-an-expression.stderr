error: expected an expression as argument

         = help: example syntax: `#[precondition(argument % 2 == 0)]`

 --> $DIR/precondition-is-not-an-expression.rs:4:15
  |
4 | #[precondition(struct)]
  |               ^^^^^^^^
